src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -134,6 +133,8 @@ export default class WaypointModel {
          */
         this.isHold = false;
 
+        // TODO: This should be moved to the StandardRouteWaypointModel
```

Comment:
I thought the WaypointModel _extended_ the StandardRouteWaypointModel, since apparently the latter is created first, from which the former is generated... but that's by passing it's own properties to the constructor (which seems fishy?). Thus, this todo is invalid, as the property _is_ in fact needed in both objects. TODO removed.